config/manager/manager.yaml

```diff
@@ -33,7 +33,7 @@ spec:
         - /usr/local/bin/manager
         args:
         - --leader-elect
-        - '-v=2'
+        - --v=0
```

Comment:
This must be `2`. What happened? The generator is enforcing it? 🤔

Reply:
@ricardozanini  Do you want INFO instead of ERROR as a default LOG level in the operator ?